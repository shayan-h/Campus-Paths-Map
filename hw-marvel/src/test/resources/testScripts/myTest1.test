CreateGraph g

AddNode g n
AddNode g n1

FindPath g x n1

FindPath g l s