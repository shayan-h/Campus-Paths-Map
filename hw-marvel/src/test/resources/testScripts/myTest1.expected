created graph g

added node n to g
added node n1 to g

unknown: x

unknown: l
unknown: s